downloader/artifact_downloader.go

```diff
@@ -147,16 +149,16 @@ func (ad *ConcurrentArtifactDownloader) download(jobs <-chan downloadJob, result
 
 // extractArchive extracts an archive using the tar CLI tool by piping the archive to the command's input.
 func (ad *ConcurrentArtifactDownloader) extractArchive(r io.Reader, targetDir string) error {
-	tarArgs := []string{
-		"-x",      // -x: extract files from an archive: https://www.gnu.org/software/tar/manual/html_node/extract.html#SEC25
-		"-f", "-", // -f "-": reads the archive from standard input: https://www.gnu.org/software/tar/manual/html_node/Device.html#SEC155
-	}
-	cmd := ad.CommandFactory.Create("tar", tarArgs, &command.Opts{
+	cmd := ad.CommandFactory.Create("unzip", []string{}, &command.Opts{
 		Stdin: r,
```

Comment:
Also it seems unzip doesn't support reading the archive from standard input.

From `man unzip`:

```
...
Archives read from standard input are not yet supported, except with funzip (and then only the first
       member of the archive can be extracted).
...
```

Reply:
a6e05c00863b4b4d060b2e80f43be7915b73ad9d